docs/tutorials/send-payjoin-with-pdk.md

```diff
@@ -0,0 +1,311 @@
+    ...
+    let mut outputs = HashMap::with_capacity(1);
+    outputs.insert(receiver_address.to_string(), amount_to_send);
+    let options = bitcoincore_rpc::json::WalletCreateFundedPsbtOptions {
+        lock_unspent: Some(false),
+        fee_rate: Some(bitcoincore_rpc::bitcoin::Amount::from_sat(10000)),
+        ..Default::default()
+    };
+    let sender_psbt = sender_wallet
+        .wallet_create_funded_psbt(
+            &[], // inputs
+            &outputs,
+            None, // locktime
+            Some(options),
+            None,
+        )
+        .unwrap();
+    let psbt = sender_wallet
+        .wallet_process_psbt(&sender_psbt.psbt, None, None, None)
+        .unwrap()
+        .psbt;
+    let psbt = Psbt::from_str(&psbt).unwrap();
+}
+```
+
+* Construct the HTTPS request
+
+  Prepare the request we want to send and also get the context back so we can
+  handle the response when its recieved.
+  
+```rust
+...
+pub async fn try_payjoin(pj_uri: payjoin::Uri<'static, NetworkChecked>) -> Txid {
+    ...
+    let (req, ctx) = RequestBuilder::from_psbt_and_uri(psbt, pj_uri)
+        .unwrap()
+        .build_with_additional_fee(
+            bitcoincore_rpc::bitcoin::Amount::from_sat(1),
+            None,
+            bitcoincore_rpc::bitcoin::FeeRate::MIN,
+            true,
+        )
+        .unwrap()
+        .extract_v1()
+        .unwrap();
+}
+```
+
+* Send the request 
+```rust
+...
+pub async fn try_payjoin(pj_uri: payjoin::Uri<'static, NetworkChecked>) -> Txid {
+    ...
+    let res = https_agent()
+        .post(req.url.as_str()) // pj_uri.endpoint
+        .header("Content-Type", "text/plain")
+        .body(String::from_utf8(req.body).unwrap())
+        .send()
+        .await
+        .unwrap();
+    let res = res.text().await.unwrap();
+}
+```
+
+* Process the response
+  
+  Validate PSBT inputs, outputs and fees as it may have been altered by the
+  reciever.
+
+```rust
+...
+pub async fn try_payjoin(pj_uri: payjoin::Uri<'static, NetworkChecked>) -> Txid {
+    ...
+    let psbt = ctx.process_response(&mut res.as_bytes()).unwrap();
+
+}
+```
+
+* Sign and finalize the Payjoin Proposal PSBT
+
+  In the previous step we validated the PSBT and now its time finish the
+  transaction, sign the PSBT and broadcast it.
+
+```rust
+...
+pub async fn try_payjoin(pj_uri: payjoin::Uri<'static, NetworkChecked>) -> Txid {
+    ...
+    let psbt = sender_wallet
+        .wallet_process_psbt(&base64::encode(psbt.serialize()), None, None, None)
+        .unwrap()
+        .psbt;
+    let tx = sender_wallet
+        .finalize_psbt(&psbt, Some(true))
+        .unwrap()
+        .hex
+        .unwrap();
+    let txid = sender_wallet.send_raw_transaction(&tx).unwrap();
+    txid
+}
+```
+* And lets call the sender functions in the `main`.
+
+  In the code below, `args` also read from the command line to consume the URI
+  and utilise `payjoin::Uri::try_from` to convert the string to a valid URI.
+
+
+```rust
+#[tokio::main]
```

Comment:
You can use `reqwest::blocking::Client` to avoid the complexity of an async runtime with reqwest's `blocking` feature